<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputGUI</class>
 <widget class="QWidget" name="OutputGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1700</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OutputGUI</string>
  </property>
  <widget class="QLabel" name="tirePhoto">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>20</y>
     <width>800</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="detectPhoto">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>20</y>
     <width>800</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="decisionText">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>850</y>
     <width>1151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
